ios/RNDateTimePickerManager.m

```diff
@@ -102,6 +119,11 @@ + (NSString*) datepickerStyleToString: (UIDatePickerStyle) style  API_AVAILABLE(
     });
 }
 
+RCT_EXPORT_SHADOW_PROPERTY(date, NSDate)
```

Comment:
Properties that might influence layout need to be passed to the shadow view as well.